### PR TITLE
Feat: add home recipe icon nav to recipe page

### PR DESCRIPTION
**Pull request by GOECHA**
**What’s this PR do?**

- Uses random icons to navigate user to view recipe page that correlates with chosen icon image

**Where should the reviewer start?**
 
- Should start on main page & click an icon image

**How should this be manually tested?**

- pull down PR and input npm start in terminal and open http://http://localhost:8080/ in browser

**Any background context you want to provide?**

- needs refactor to dry up code

### CODE REVIEW

File: src/scripts.js [hunk 1]
```diff
@@ -133,7 +136,8 @@ function showFavoritesPage() {
 }
 
 function viewRecipe(event) {
-  selectedRecipeName = event.target.innerText;
+  selectedRecipeName = event.target.innerText
+  console.log(selectedRecipeName)
```
Comment (tfettig22) [inline src/scripts.js:140]: need to remove console.log - fine for now but just to remember for future pushes

File: src/scripts.js [hunk 2]
```diff
@@ -149,6 +153,26 @@ function viewRecipe(event) {
   displaySelectedRecipeImg();
 }
 
+function viewRecipeFromIcon(event){
```
Comment (tfettig22) [inline src/scripts.js:156]: Nice job getting this function working! I'd love to try and see if we can DRY it up from the function above it, but for now it is good and most important part is that it's working!

File: src/scripts.js [hunk 3]
```diff
@@ -245,6 +269,7 @@ function filterFavoriteRecipesByName(name) {
 }
 
 function showFavoriteRecipeImages(){
+  show(favoriteRecipeImages);
```
Comment (tfettig22) [inline src/scripts.js:272]: do we want this here?

**Review by tfettig22 — APPROVED** [review]
May have some refactoring to do later but for now this looks great, thank you!

**Review by forsethnico — APPROVED** [review]

**Pull request merged**